src/main/java/Floda.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
```

Comment:
I like how you specified java.util.Scanner instead of java.util.* , this complies with coding standards, well done !